config/routes.rb

```diff
@@ -12,6 +14,7 @@ def call(env)
   end
 end
 
+# rubocop:disable Metrics/BlockLength
```

Comment:
In this case, this comment is good, because this is just how `routes.rb` is like, and there isn't much to do about it.
👍🏼